pontoon/base/models.py

```diff
@@ -2248,13 +2248,24 @@ def reset_active_translations(self, locale):
 
         translations.filter(pk__in=unreviewed_pks).update(active=True)
 
+    def _get_word_count(self, string):
+        return len(re.findall(r'\w+', string))
```

Comment:
I'd simplify this to what @Pike does in [`compare-locales`](https://github.com/Pike/compare-locales/blob/c3c59dae92a717014c23c4708675a4555064f6fa/compare_locales/parser/fluent.py#L37):
```python
return len(string.value.split())
```

Reply:
@mathjazz ok, I'll change it. 

Just want to notice that cases with dash will be counted in a wrong way. 
e.g. My brothers—Richard and John—are visiting Hanoi. 
using split method get_word_count will count 'brothers—Richard' as one word. 
or in case with spaces 'brothers — Richard' as three words.

All separately stand digits and symbols will be count as word as well(here is hard to understand if it's an expected result or no).

Reply:
Good point. I think the first example lacks a space, but the 2nd example seems legit.

@Pike I'd like to keep the word count method consistent with [`compare-locales`](https://github.com/Pike/compare-locales/blob/c3c59dae92a717014c23c4708675a4555064f6fa/compare_locales/parser/fluent.py#L37), which in the example given by @Karska-dev counts `—` as words I believe. Are we fine with that?

Reply:
Seems that there isn't just one answer, and the c-l wordcount isn't right. flod found
http://unicode.org/reports/tr29/#Word_Boundaries and https://docs.sdl.com/LiveContent/content/en-US/SDL%20WorldServer-v7/GUID-376E123B-1C7E-4D64-82B0-1D33F088ABD5#addHistory=true&filename=GUID-20199935-0650-461B-99E1-F9C45FCC8A43.xml&docid=GUID-20199935-0650-461B-99E1-F9C45FCC8A43&inner_id=&tid=&query=&scope=&resource=&toc=false&eventType=lcContent.loadDocGUID-20199935-0650-461B-99E1-F9C45FCC8A43

Jeff said that he'd like hyphenated words to be one.

My suggestion would be to use `r'[\w,.-]+'`

Reply:
Done